direct/src/dist/locations.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+if sys.version_info < (3, 10):
+    import distutils.sysconfig as sysconf
+    PythonIPath = sysconf.get_python_inc()
+    PythonVersion = sysconf.get_config_var("LDVERSION") or sysconf.get_python_version()
+    Python = sysconf.PREFIX
+    lib_dir = sysconf.get_python_lib(plat_specific=1, standard_lib=1)
+    config_vars = sysconf.get_config_vars()
+else:
+    import sysconfig as sysconf
+    PythonIPath = sysconf.get_path("include")
+    PythonVersion = sysconf.get_config_var("LDVERSION") or sysconf.get_python_version()
+    Python = sysconf.get_config_var('prefix')
```

Comment:
Not a fan of the naming of these variables

Reply:
I don't either. I would prefer this file to just be a super thin wrapper for `distutils.sysconf`, even keeping the original names:

```python
try:
  from distutils.sysconfig import *

 # not sure if this works or if we need to setup explicit aliases
  __all__ = ['get_python_inc', 'get_config_var', ...] 
except ImportError:
  from sysconfig  import *

  __all__ = ['get_config_var', ...]

  def get_python_inc():
    return get_path('include')
```